### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by thepomeranian**
* Renamed Token Objects to Digital Assets
* Authors must now create their own GH Issue for discussion
* Added AIP-43 and AIP-44
* Added new AIP state `Ready for Review`
* Introduce Gas, Cryptography, Interface, and Platform Categories
* Added notice regarding rejected or stale AIPs
* Added notice regarding renaming AIPs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,15 +83,27 @@ This table contains an overview of all created and tracked AIPs. It should be up
...
 | `Rejected` | A community decision has been made to not move forward with an AIP at this time.| 
 | `On Hold` | Some information is missing or prerequisites have not yet been completed. | 
 
 ## Layer
 * Framework
 * Core (Blockchain)
+* Gas
+* Cryptography
+* Interface
+* Platform
```
Comment (davidiw) [inline README.md:97]: @gregnazario , @banool can you pick one? I don't know if we need both of these?

File: README.md [hunk 2]
```diff
@@ -36,18 +37,18 @@ This table contains an overview of all created and tracked AIPs. It should be up
 | [AIP 8](https://github.com/aptos-foundation/AIPs/issues/33) | Framework | [Higher-Order Inline Functions for Collections](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-8.md) | Standard | Accepted |
 | [AIP 9](https://github.com/aptos-foundation/AIPs/issues/26) | Interface/ Framework | [Resource Groups](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-9.md) | Standard | Accepted
 | [AIP 10](https://github.com/aptos-foundation/AIPs/issues/27) | Framework | [Move Objects](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-10.md) | Standard | Accepted | 
-| [AIP 11](https://github.com/aptos-foundation/AIPs/issues/31) | Framework | [Tokens as Objects](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-11.md) | Standard | Accepted |
+| [AIP 11](https://github.com/aptos-foundation/AIPs/issues/31) | Framework | [Digital Assets (Tokens as Objects)](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-11.md) | Standard | Accepted |
```
Comment (davidiw) [inline README.md:40]: Probably "Digital Assets: Tokens as Objects"

File: README.md [hunk 3]
```diff
@@ -68,6 +69,8 @@ This table contains an overview of all created and tracked AIPs. It should be up
 | [AIP 40](https://github.com/aptos-foundation/AIPs/issues/178)| Core | [Address Standard v1](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-40.md) | Standard | Draft |
 | [AIP 41](https://github.com/aptos-foundation/AIPs/issues/185)| Framework | [Move module for randomness generation](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-41.md) | Standard | Draft |
 | AIP 42 | Core | [Aptos TypeScript SDK V2](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-42.md) | Standard | Draft |
+| AIP 43 | Framework | [Token V2 Throughput Improvement](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-43.md) | Standard | Draft |
```
Comment (davidiw) [inline README.md:72]: Probably Token Objects or Digital Asset

**Review by davidiw — COMMENTED** [review]
otherwise excellent!

File: README.md (hunk 1 above)
Comment (gregnazario) [reply README.md:97]: Let's go with Platform

**Review by davidiw — APPROVED** [review]

**Review by movekevin — APPROVED** [review]

**Pull request merged**